exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
> Any particular reason to go with `subject`?

This is another testing habit, I always call the object under test `subject` which makes it clear which object is being tested. It just happens here that `subject` is a possible domain specific word so changing it to `school` for this test file is probably a good idea.